microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -23,62 +23,63 @@ ms.date: 02/24/2023
    limitations under the License.  -->
 # Emulate and test other browsers
 
-Your job doesn't end with making sure your site runs great across Microsoft Edge and Android.  Even though the [Device Emulation](index.md) tool can simulate a range of other devices such as smart phones, we encourage you to check out solutions for emulation provided by other browsers.
+To make sure your website works for all your users, test your website on other browsers and devices. Not all of your users will be using the browser and device that you used when developing. We encourage you to check out solutions for emulation provided by other browsers, in addition to using the **Device Emulation** tool in Microsoft Edge DevTools.
 
 
 <!-- ------------------------------ -->
 #### Summary
 
-*  When you don't have a particular device, or want to do a spot check on something, the best option is to emulate the device right inside your browser.
+There are several different levels of testing that you can do to test your website under different conditions:
 
-*  Device emulators and simulators enable you to mimic your development site on a range of devices, from your workstation.
+* **Browser emulators** help you to quickly test how your website looks and behave at different screen sizes and resolutions, with touch event emulation, and by overriding the User-Agent string that the browser sends. You use browser emulators from the browser you're developing on. See [Browser emulators](#browser-emulators), below.
```

Comment:
```suggestion
* **Browser emulators** help you to quickly test how your website looks and behaves at different screen sizes and resolutions, with touch event emulation, and by overriding the User-Agent string that the browser sends. You use browser emulators from the browser you're developing on. See [Browser emulators](#browser-emulators), below.
```

Reply:
**that** you're developing on.  will push commit:  (behaves)
```
* **Browser emulators** help you to quickly test how 
your website looks and behaves at different screen sizes 
and resolutions, with touch event emulation, and by 
overriding the User-Agent string that the browser sends.  You 
use browser emulators from the browser that you're 
developing on.  See [Browser emulators](#browser-emulators), 
below.
```